Rakefile

```diff
@@ -77,19 +88,29 @@ task :lint do
   sh "npm run-script lint"
 end
 
-desc "Run webpack with development settings"
-task :webpack do
-  sh "BUILD_TYPE=dev npm run-script webpack"
+desc "Run webpack with development features turned on. Build is optimized."
+task :webpack_dev do
+  sh "rm -rf assets/js/generated"
+  if enable_optimizations then
+    sh "BUILD_TYPE=dev npm run-script webpack"
+  else
+    sh "BUILD_TYPE=dev npm run-script webpack-noopt"
```

Comment:
Assuming `noopt` stands for `no-optimization`?  If this is the case, why is it `noopt` instead of `no-opt`?